### ISSUE TITLE
Web to Prospect API Problem

### ISSUE THREAD

**Issue by thomastsaidotcom** [propertybase/propertybase-website-demo#2]
I set up the demo install at http://arctest.outerspacehosting.com

The Search Request form at the bottom of the home page, when filled out, will denote success, but no contact gets created in propertybase.

**Comment by PeterSiegl** [comment]
Sorry for the late answer, I somehow missed your message!
Did you add your web to prospect endpoint and token to the config.php? Please let me know if the issue persists!

**Comment by thomastsaidotcom** [comment]
Yes, my web to prospect endpoint and token are in config.php. It wouldn’t
give me a success message on submitting the form otherwise. Nonetheless, I
just tested it again and the issue is persisting.

On October 5, 2016 at 6:52:53 AM, Peter (notifications@github.com) wrote:

Sorry for the late answer, I somehow missed your message!
Did you add your web to prospect endpoint and token to the config.php?
Please let me know if the issue persists!

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
https://github.com/propertybase/propertybase-website-demo/issues/2#issuecomment-251680898,
or mute the thread

**Comment by PeterSiegl** [comment]
I suppose the request on listings works? I just tested the search request form on your demo page and I don't get any errors in the console, so it should work just fine.

Inside Propertybase did you check the display `All` list on the request object?

**Comment by thomastsaidotcom** [comment]
The request on the listing works fine. However, the Search Request form
does not work.

Yes, inside propertybase I did check the display all list.

On October 6, 2016 at 9:41:37 AM, Peter (notifications@github.com) wrote:

I suppose the request on listing works? I just tested the search request
form on your demo page and I don't get any errors in the console, so it
should work just fine.

Inside Propertybase did you check the display All list on the request
object?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
https://github.com/propertybase/propertybase-website-demo/issues/2#issuecomment-252019696,
or mute the thread
